service-front/module/Application/view/application/pages/how_will_the_donor_confirm.twig

```diff
@@ -12,9 +12,10 @@
             Confirming their identity over the phone is the quickest and easiest option - they'll need an NI number,
             passport or driving licence and to answer some identity check questions during the call.
 
-            <br><br>Alternately, they can identify in-person at a Post Office - this will take longer and incurs a
-            cost
-            to OPG.
+            <br><br>
```

Comment:
If you are adding `<br>`s to visually space things, consider if these should be new paragraph/`<p>` elements instead.